src/core/dbsetup.py

```diff
@@ -0,0 +1,13 @@
+import os
+import mysql.connector
+from dotenv import load_dotenv
+
+load_dotenv()
+
+ACCOUNT_PASSWORD = os.getenv('LOCALHOST_PASSWORD')
```

Comment:
LOCALHOST_PASSWORD should be DB_PASSWORD.. same for username

Reply:
same for host